bundles/com.espressif.idf.core/src/com/espressif/idf/core/build/messages.properties

```diff
@@ -32,10 +32,10 @@ NoActiveEspIdfInWorkspaceMsgTitle=ESP-IDF Setup
 NoActiveEspIdfInWorkspaceMsg=ESP-IDF is required to use Espressif IDE. Would you like to configure it now?
 
 OldConfigFoundMsgBoxTitle=Old Configuration Found
-OldConfigFoundMsgBoxMsg=An old ESP-IDF configuration has been detected.\nIf you want to use these you need to export these configuration and pass them on to EIM for updated configurations.\nWould you like to export these settings?\n\nPress 'Yes' to choose a location to save.
+OldConfigFoundMsgBoxMsg=Please export the current workspace configuration to the EIM configuration for proper environment setup. Do you want to export configurations?
```

Comment:
Can we convey below points in the message?
- Explain why the prompt is appearing (legacy setup detected).
- Clarify what will happen (conversion to EIM format).
- State how it benefits the user (proper management and IDE will be ready to use with the existing setup).

Something like below or whatever you think that will convey above points - 
> Espressif IDE now uses EIM to manage ESP-IDF Installations. A legacy ESP-IDF configuration was detected in your workspace. Would you like to convert it to the EIM-compatible format to prepare the IDE for use?